frontend/src/components/LinkInformations/RedeemCommunitySelection.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <div
+    :link-data="linkData"
+    :redeem-code="redeemCode"
+    :is-contribution-link="isContributionLink"
+    :is-redeem-jwt-link="isRedeemJwtLink"
+    class="redeem-community-selection"
+  >
```

Comment:
What is the function, the reason of that? Div cannot do any thing with the parameters, only showing in browser:
```html
<div link-data="[object Object]" redeem-code="6cf60204cd170196d2c8bdef" is-contribution-link="false" is-redeem-jwt-link="false" class="redeem-community-selection" is-transaction-link-loaded="true">
```